src/cr/cube/dimension.py

```diff
@@ -954,6 +959,41 @@ def _order_dict(self):
         return self._dimension_transforms_dict.get("order") or {}
 
 
+class _PairwiseSignificanceSpec(object):
+    """Value object providing access to details of pairwise significance information."""
```

Comment:
Okay, I'm still getting used to the idea of this being a separate value object rather than just being attributes directly on `Dimension`, but I think it's probably the right choice.

In any case, why wouldn't you move `Cube.alpha_values`, `.only_larger`, etc. down here? You could still get them temporarily from a dimension (which `_Slice` has access to) if they're necessary during the transition and then you wouldn't need the `CubePartition._alpha` and `._alpha_alt`, etc. properties at all, you could just inline the access to these in the pairwise-sig implementations like that of `_Slice.summary_pairwise_indices` that were going away anyway.

Reply:
gothca ... thanks nice idea